app.js

```diff
@@ -1,16 +1,162 @@
 /* Imports */
 
 /* State */
+let pick = '';
+let winCount = 0;
+let loseCount = 0;
+let tieCount = 0;
+
+let fight1 = document.getElementById('fight1');
+let fight2 = document.getElementById('fight2');
+let fight3 = document.getElementById('fight3');
+
+let fighterPick = document.getElementById('fighter-select');
+let sbtxt = document.getElementById('subtext');
+
+let play1 = document.getElementById('play1');
+let play2 = document.getElementById('play2');
+let play3 = document.getElementById('play3');
+
+let compPlay1 = document.getElementById('compplay1');
+let compPlay2 = document.getElementById('compplay2');
+let compPlay3 = document.getElementById('compplay3');
+
+const fighters = ['link', 'cucco', 'bokoblin'];
+let compPick = getRandomItem(fighters);
+let playAgain = document.getElementById('playagain');
+
+let winDisp = document.getElementById('total-wins');
+let loseDisp = document.getElementById('total-losses');
+let tieDisp = document.getElementById('total-ties');
 
 /* Actions */
+function getRandomItem(array) {
+    const random = getRandomNumber(array.length);
+    const item = array[random];
+    return item;
+}
+function getRandomNumber(choices) {
+    return Math.floor(Math.random() * choices);
+}
+
 function loadPage() {}
 
+function fightPick(userPick) {
+    fighterPick.classList.add('hide');
+    pick = userPick;
+    dispFighter();
+    compPick;
```

Comment:
Right here you are just storing the value of `compPick` but you aren't doing anything with it.